br/tests/lightning_extend_routes/config.toml

```diff
@@ -0,0 +1,23 @@
+
+[mydumper]
+source-id = "mysql-01"
+
+# the complicated routing rules should be tested in tidb-tools repo already
+# here we're just verifying the basic things do work.
+[[routes]]
+schema-pattern = "routes_a*"
+table-pattern = "t*"
+target-schema = "routes_b"
+target-table = "u"
+
+    [routes.extract-table]
+    table-regexp = "t(.*)"
```

Comment:
Does this mean Lightning will evict the prefix of the table name, e.g. `t` in this case, and insert the rest as the value of the extra column?

If so, will it be better to use a suffix expression like 't*'?

Reply:
This means cut characters after t like cut "1" from "t1". The target-table is set by `routes.target-table`

Reply:
If it designs to truncate the head of the `table-regexp`, which looks quite ambiguous, why not use suffix expression like `t*`. 

For example, if the user input: `t[0-9A-Z]+abc.*`, what should be truncated and what is ought to be left?

Reply:
the routed result is not generated by truncate/left. It's decided by `target-table`.